### DISCUSSION TITLE
Do we have a white text on black screen theme?

### DISCUSSION THREAD

**Discussion by grv07** [helix-editor/helix#12370] [Q&A] [answered]
I suddenly lost my interest in all color themes, I would like a white text on black screen.

**Comment by JeftavanderHorst** [comment]
I'm not aware of a built-in theme, but you could update one of these to only have two colors: https://github.com/helix-editor/helix/pull/11872

**Comment by grv07** [comment, marked as answer]
@JeftavanderHorst 
I have created my own [https://github.com/grv07/white-dark](url)
